### ISSUE TITLE
Create `pl-jsplumb` element

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#911]
jsplumb https://github.com/jsplumb/jsplumb
3js https://threejs.org/

**Comment by mwest1066** [comment]
Looks like threejs got implemented, but jsplumb still looks cool for things like finite state machines.

**Comment by nwalters512** [comment]
@eliotwrobson does this overlap with your work on a finite state machine element? Did you consider using jsplumb for that?

**Comment by eliotwrobson** [comment]
Not quite. The current FSM builder element uses vanilla JavaScript taken from another open source library. There could be a way to integrate it with this proposed element, but it's probably not worth the extra effort at this point.

**Comment by eliotwrobson** [comment]
Is there still desire for this? the threejs element was deprecated, I'm not sure if a jsplumb element would be worth it. We can always reopen if people ask for it.
